optimum/exporters/onnx/model_configs.py

```diff
@@ -2648,3 +2649,33 @@ def outputs(self) -> Dict[str, Dict[int, str]]:
 
 class PatchTSMixerOnnxConfig(PatchTSTOnnxConfig):
     pass
+
+
+class RTDetrOnnxConfig(ViTOnnxConfig):
+    # Export the operator 'aten::grid_sampler' to ONNX fails under opset 16.
+    # Support for this operator was added in version 16.
+    DEFAULT_ONNX_OPSET = 16
+    ATOL_FOR_VALIDATION = 1e-5
+
+    @property
+    def inputs(self) -> Dict[str, Dict[int, str]]:
+        return {
+            "pixel_values": {0: "batch_size", 2: "height", 3: "width"},
+        }
+
+    def _create_dummy_input_generator_classes(self, **kwargs) -> List["DummyInputGenerator"]:
+        if kwargs["height"] < 320:
+            warnings.warn(
+                f"Exporting model with image `height={kwargs['height']}` which is less than minimal 320, setting `height` to 320."
+            )
+            kwargs["height"] = 320
+        if kwargs["width"] < 320:
+            warnings.warn(
+                f"Exporting model with image `width={kwargs['width']}` which is less than minimal 320, setting `width` to 320."
+            )
+            kwargs["width"] = 320
```

Comment:
why is it needed and where does the value 320 comes from ?

Reply:
default height and width is set to 64, but export does not work for those values, so I override it with minimum value divisible by 32 which is supported for export (should be grater that 300)

Reply:
Thanks for the explanation! In this case shouldn't we check for `num_queries` in the config directly as this could vary between models ? (can be set to any value by default in cases this value cannot be extracted from the `self._config`)

Reply:
Sounds good, I've updated the implementation

Reply:
perfect thanks @qubvel !